### ISSUE TITLE
Move Gnuplot export out of BaseMass.h 

### ISSUE THREAD

**Issue by damienmarchal** [sofa-framework/sofa#3443]
These functions have nothing to do there and must go into a GnuPlotExporter or something like that.

**Comment by alxbilger** [comment]
I agree. And this has to be done also in `BaseInteractionForceField` and `BaseInteractionForceField`
